includes/wizards/newspack/class-newspack-dashboard.php

```diff
@@ -129,7 +129,11 @@ public function get_dashboard() {
 					],
 				],
 			],
-			'listings'             => [
+		];
+
+		// Newspack Listings Plugin.
+		if ( is_plugin_active( 'newspack-listings/newspack-listings.php' ) ) {
```

Comment:
```suggestion
		if ( defined( 'NEWSPACK_LISTINGS_FILE' ) ) {
```

Same as the other comment.

Reply:
Agreed!